README.md

```diff
@@ -1,52 +1,80 @@
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-- Here is the page preview
+<!-- PROJECT DESCRIPTION -->
 
-> |     | Mobile version                                  | Desktop version                                  |     |
-> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
-> |     | ![Screenshot1](./images/screenshots/mobile.png) | ![Screenshot2](./images/screenshots/desktop.png) |
+# 📖 [GLOBAL REMOTE SOFTWARE DEVELOPER SUMMIT 2023] <a name="about-project"></a>
 
-This project is to put to test the understanding of individual student on what they've studied so far from the curriculum's module 1.
+> This website is the result of the first Capstone Project for the Microverse Program😊. This is showing the World Tour for La Rienda Band, an argentinian band that will travel around the world.👌
 
-> Learning objectives
+## 🛠️ Built With <a name="built-with"></a>
 
-- I Used semantic HTML tags.
-- I Applied best practices in HTML code.
-- Used CSS selectors correctly.
-- Used CSS box model.
-- Used Flexbox to place elements on the page.
-- Demonstrated ability to create UIs adaptable to different screen sizes using media queries.
-- Used GitHub Pages to deploy web pages.
-- Applied JavaScript best practices and language style guides in code.
-- Used JavaScript to manipulate DOM elements.
-- Used JavaScript events.
-- Used objects to store and access data.
-- Communicated technical concepts to other technical people.
+### Tech Stack <a name="tech-stack"></a>
 
-## Built With
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/TR/2011/WD-html5-20110405/">HTML5</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/specs.en.html">CSS</a></li>
+    <li><a href="https://www.ecma-international.org/publications-and-standards/standards/ecma-262/">JavaScript</a></li>
+  </ul>
+</details>
 
-- HTML
-- CSS
-- JAVASCRIPT
+<!-- Features -->
 
-Click [here]() to watch a walk-through description, given by me.
+### Key Features <a name="key-features"></a>
 
-## Additional Tools
+- Landing Page for activities
+- Mobile first development
+- Responsiveness and flexibility for any screen size
+- Apply JavaScript best practices and language style guides in code.👌
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.💯
+- Use objects to store and access data.
+- Communicate technical concepts to other technical people.
 
-- WebHint
-- Stylelint
-- ESlint
-- Node Packet Manager
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Live Demo
+<!-- LIVE DEMO -->
 
-[Here]() is the live demo.
+## 🤯 Walk through video <a name="walk-through"></a>
 
```

Comment:
- [ ] You did a great job at the presentation of the project and you did 😊⤵️

## Highlights 🚀👌
1. Recorded a video presenting the features of the project.
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a demo of the project, showing the pages in different screen sizes.

## Forgotten Things 🍀
- [ ] But I see that you forgot to : 
1. Show some piece of your code as it is from the requirements, so please 😊The video should contain a highlight about a piece of the code the student finds interesting or is proud of making, and with that, your presentation will be great and professional 👌😊
2. The maximum length of the video is 5 minutes, but in your case, I see that you exceeded 5 minutes for the video, so please try to record the maximum length of the video is 5 minutes to stick to the requirements